cli/src/service.rs

```diff
@@ -355,84 +359,212 @@ pub fn new_light(config: Configuration) -> Result<impl AbstractService, ServiceE
 
 #[cfg(test)]
 mod tests {
-
-	#[cfg(feature = "rhd")]
+	use crate::service::{new_full, new_light};
+	use cennznet_primitives::types::{Block, DigestItem, Signature};
+	use cennznet_runtime::constants::{asset::SPENDING_ASSET_ID, currency::CENTS, time::SLOT_DURATION};
+	use cennznet_runtime::{Address, Call, GenericAssetCall, UncheckedExtrinsic};
+	use codec::{Decode, Encode};
+	use sc_consensus_babe::{BabeIntermediate, CompatibleDigestItem, INTERMEDIATE_KEY};
+	use sc_consensus_epochs::descendent_query;
+	use sc_service::AbstractService;
+	use sp_consensus::{
+		BlockImport, BlockImportParams, BlockOrigin, Environment, ForkChoiceStrategy, Proposer, RecordProof,
+	};
+	use sp_core::{crypto::Pair as CryptoPair, H256};
+	use sp_finality_tracker;
+	use sp_keyring::AccountKeyring;
+	use sp_runtime::traits::IdentifyAccount;
+	use sp_runtime::{
```

Comment:
Squash `traits::IdentifyAccount` inside here?

Reply:
It's matching the corresponding file in Plug. We'd better not to make such changes in cennznet to keep it comparable with the source for future updates.

Reply:
May need to check with tech leads on code standards to get the team on the same page. cc @holygits